src/main/java/org/sagebionetworks/bridge/spring/controllers/AccountsController.java

```diff
@@ -222,21 +227,29 @@ public StatusMessage signOut(@PathVariable String userId,
         App app = appService.getApp(session.getAppId());
         participantService.signUserOut(app, userId, deleteReauthToken);
 
-        return new StatusMessage("User signed out.");
+        return SIGN_OUT_MSG;
     }
     
-    public Account verifyOrgAdminIsActingOnOrgMember(String appId, String orgId, String userId) {
-        // The caller needs to be an administrator of this organization
-        if (!IS_ORGADMIN.check(ORG_ID, orgId)) {
-            throw new EntityNotFoundException(Account.class);
+    /**
+     * This kind of code is driving me nuts... I'm not sure how to rationalize it further. We need
+     * to retrieve the account and test its organization membership. 
+     */
+    public Account verifyOrgAdminIsActingOnOrgMember(String appId, String callerOrgId, String userIdToken) {
+        // The caller needs to be associated to an organization
+        if (callerOrgId == null) {
+            throw new UnauthorizedException();
```

Comment:
Because this is an API for admin accounts, I'm assuming admins need to create the account in an organization. The account could be picked up through the unassigned API, but it's better if the account is immediately in an organization. If we accounts in multiple organizations or teams, this might need to be rethought.

Reply:
Admins and superadmins will no longer have access to these APIs. This could be problematic if admins and superadmins need access these APIs for administrative or debugging purposes.

Reply:
Admins should be in an organization (despite the fact that they can access everything in an app, all administrative accounts would/should be associated to an organization), but super admins do not have to be (indeed they can't because the organizations are different between apps). Okay I will add an exception here for super admins.

Reply:
No never mind, admins should *also* be able to retrieve accounts outside of their organization, adding that as well.